maintenance/idr0021-data-prep.md

```diff
@@ -0,0 +1,76 @@
+
+Download and import idr0021 data
+================================
+
+To download IDR data and re-import elsewhere....
+(See gdoc https://docs.google.com/document/d/1TmBZ43_yhiO3AOua8oMk4mPWKWJtpeYNc2KLP17h-1I/edit# )
+
+	$ ssh idr0-slot3.openmicroscopy.org
```

Comment:
Since we do not offer the data publicly. The step should be added to the internal doc and here only indicate "contact us for data access" or something like that